packages/venia-concept/src/components/MiniCart/productList.js

```diff
@@ -1,56 +1,50 @@
-import React, { Component } from 'react';
-import { arrayOf, number, shape, string } from 'prop-types';
+import React from 'react';
+import { array, func, shape, string } from 'prop-types';
 import { List } from '@magento/peregrine';
 
-import classify from 'src/classify';
+import { mergeClasses } from 'src/classify';
+
 import Product from './product';
 import defaultClasses from './productList.css';
 
-class ProductList extends Component {
-    static propTypes = {
-        classes: shape({
-            root: string
-        }),
-        items: arrayOf(
-            shape({
-                item_id: number.isRequired,
-                name: string.isRequired,
-                price: number.isRequired,
-                product_type: string,
-                qty: number.isRequired,
-                quote_id: string,
-                sku: string.isRequired
-            })
-        ).isRequired,
-        currencyCode: string.isRequired
-    };
+const ProductList = props => {
+    const {
+        beginEditItem,
+        cartItems,
+        currencyCode,
+        removeItemFromCart
+    } = props;
 
-    render() {
-        const {
-            currencyCode,
-            removeItemFromCart,
-            openOptionsDrawer,
-            totalsItems,
-            ...otherProps
-        } = this.props;
+    const classes = mergeClasses(defaultClasses, props.classes);
 
-        return (
-            <List
-                render="ul"
-                getItemKey={item => item.item_id}
-                renderItem={props => (
+    return (
+        <List
```

Comment:
It's fine to leave this here since it already existed, but going forward we should avoid using `List`—at least in this render prop form.

Reply:
Makes sense - want me to create a refactor ticket for it?

Reply:
Created #1328 , feel free to add more details or close if desired.